docs/contributing.rst

```diff
@@ -0,0 +1,60 @@
+=====================================================
```

Comment:
Docs or wiki? I think that we should encourage people back into the GitHub repo for this sort of stuff. Perhaps we should just link to our [CONTRIBUTING.md](https://github.com/pi-top/pi-top-Python-SDK/blob/master/.github/CONTRIBUTING.md) from here?

Reply:
I think having the "contributing" section in the online docs has value... And agreed, it should redirect to our repo's page